### ISSUE TITLE
Doesn't appear to be using Nvidia RTX 3060 but instead using my CPU

### ISSUE THREAD

**Issue by Firnen08** [Plachtaa/seed-vc#38]
Warnings such as this pop up in the console

N:\seed-vc-main\modules\rmvpe.py:508: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  ckpt = torch.load(model_path, **map_location="cpu")**
  
  Might have something to do with it using CPU instead of GPU. unless im doing something wrong, i have Latest CUDA installed, Latest Drivers, Etc

**Comment by Plachtaa** [comment]
run
```
import torch
print(torch.cuda.is_available()`
```
to see whether your CUDA is available or not
the code you send has nothing to do with GPU or CPU usage. Instead, model weights are intentionally loaded to cpu first, then moved to CUDA

**Comment by zhixianjuli** [comment]
You can uninstall torch torchvision torchaudio. And then install as
pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121

**Comment by Firnen08** [comment]
> You can uninstall torch torchvision torchaudio. And then install as pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121

this fixed it, forgot to respond sorry!
